spinalcordtoolbox/deepseg/models.py

```diff
@@ -19,23 +20,44 @@
 # TODO: add mirror
 MODELS = {
     't2star_sc':
-        {'url': 'https://osf.io/v9hs8/download?version=4',
-         'description': 'Cord segmentation on T2*-weighted contrast.',
+        {'url': 'https://osf.io/v9hs8/download?version=5',
+         'description': 'Cord segmentation model on T2*-weighted contrast.',
          'default': True},
     'mice_uqueensland_sc':
-        {'url': 'https://osf.io/nu3ma/download?version=5',
-         'description': 'Cord segmentation on mouse MRI. Data from University of Queensland.',
+        {'url': 'https://osf.io/nu3ma/download?version=6',
+         'description': 'Cord segmentation model on mouse MRI. Data from University of Queensland.',
          'default': False},
     'mice_uqueensland_gm':
-        {'url': 'https://osf.io/mfxwg/download?version=5',
-         'description': 'Gray matter segmentation on mouse MRI. Data from University of Queensland.',
+        {'url': 'https://osf.io/mfxwg/download?version=6',
+         'description': 'Gray matter segmentation model on mouse MRI. Data from University of Queensland.',
          'default': False},
     't2_tumor':
-        {'url': 'https://osf.io/uwe7k/download?version=1',
-         'description': 'Cord tumor segmentation on T2-weighted contrast.',
+        {'url': 'https://osf.io/uwe7k/download?version=2',
+         'description': 'Cord tumor segmentation model, trained on T2-weighted contrast.',
+         'default': False},
+    'findcord_tumor':
+        {'url': 'https://osf.io/qj6d5/download?version=1',
+         'description': 'Cord localisation model, trained on T2-weighted images with tumor.',
          'default': False}
     }
 
+# List of task. The convention for task names is: action_(animal)_region_(contrast)
+# Regions could be: sc, gm, lesion, tumor
+TASKS = {
+    'seg_sc_t2star':
+        {'description': 'Cord segmentation on T2*-weighted contrast.',
+         'models': ['t2star_sc']},
+    'seg_mice_sc':
+        {'description': 'Cord segmentation on mouse MRI.',
+         'models': ['mice_uqueensland_sc']},
+    'seg_mice_gm':
+        {'description': 'Gray matter segmentation on mouse MRI.',
+         'models': ['mice_uqueensland_gm']},
+    'seg_tumor_t2w':
```

Comment:
To be consistent with the « t2star » case
```suggestion
    'seg_tumor_t2':
```